bundles/AdminBundle/Resources/public/js/pimcore/object/folder.js

```diff
@@ -343,6 +343,22 @@ pimcore.object.folder = Class.create(pimcore.object.abstract, {
 
         this.tab.mask();
 
+        try {
+            pimcore.plugin.broker.fireEvent('preSaveObject', this, 'object');
+        } catch (e) {
+            if (e instanceof pimcore.error.ValidationException) {
+                this.tab.unmask();
```

Comment:
Just a short question. This unmasking ... shouldn't it be done for any kind of exception?

Reply:
I think yes, in this case I just copied it from https://github.com/pimcore/pimcore/blob/8b948ea0f182b250b7744122772b7f5f9f9677e6/bundles/AdminBundle/Resources/public/js/pimcore/object/object.js#L749-L763